### PR TITLE
Added an ability to use Highstock and Highmaps

### LINKED ISSUE #2: Highmaps
It would be amazing if this Ember-CLI addon included Highmaps too.  Are there any plans to include maps?  Thanks!

### PR DESCRIPTION
**Pull request by lolmaus**
Fixes #2.

I have removed addBowerPackageToProject('highcharts-release#~4.0.4') and app.import(app.bowerDirectory + '/highcharts-release/highcharts.js');.

It is up to user whether to import Highcharts, Highstock and/org Highmaps, which are distributed from separate repos. I have added instructions to the readme.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -12,9 +12,7 @@
     "ember-load-initializers": "ember-cli/ember-load-initializers#0.0.2",
     "ember-qunit": "0.1.8",
     "ember-qunit-notifications": "0.0.5",
-    "qunit": "~1.17.1"
-  },
-  "devDependencies": {
-    "highcharts-release": "~4.0.4"
+    "qunit": "~1.17.1",
+    "highstock-release": "~2.1.3"
```
Comment (Dhaulagiri) [inline bower.json]: Did you mean to leave this in as a dependency? Or should users explicitly add the package(s) they want?

Comment (lolmaus) [inline bower.json]: This is necessary for building the demo and running tests. It doesn't get into users' apps.

**Comment by ahmadsoe** [comment]
@lolmaus can we make the installation process more simple? leave the charts as default, and if users want highstock or highmaps they just need to put some config on Brocfile or enviroment.js.

**Comment by lolmaus** [comment]
@ahmadsoe, we can have `ember-highcharts` update `bower.json` during installation.

But importing the Bower package is a problem. If we let `ember-highcharts` do it, there is no way for the user to import a different one.

To mitigate that we'll have to introduce some kind of configuration variable accessible from the Node side of Ember CLI. I am not familiar enough with Ember CLI to do that.

And i believe having a `bower.json` update without automatically importing the package is not consistent.

* * *

I'm not trying to press you, but if you don't accept this, i'm gonna publish my fork as a separate package. I'm gonna try introducing basic dynamic data (#1) and i'll do it in my fork if our ways part. I will of course use a different package name and leave proper credit in the readme.

I hate community splits, so i would prefer to work in your package. Can you please accept this PR, sacrificing some installation convenience? And then, when somebody figures out how to conditionally import a Bower package, they do another PR restoring the convenience?

**Comment by ahmadsoe** [comment]
Who says I am not gonna merge this? This is a good start to including highstocks and highmaps components on this addon.

**Comment by ahmadsoe** [comment]
Merged.. thanks @lolmaus

**Pull request merged**